### PR TITLE
#4918 cookie banner

### LINKED ISSUE #4918: Cookies popup design
image
- compromise between the default cookie popup and the full version ( with accept, policy and decline visible right away) 
- lets see how this performs, we can change it anytime

- cookie popup is displayed in the left side of screen, in case of mobile its middle

- when you click the 3 dots, dropdown is displayed - our policy and Reject all 

- accept all has the normal hover effect, in light mode - k-hovergrey, in dark mode k-shade

- top version is for desktop, under which is mobile version, hope it's not complicated much, lmk

- design in figma - landing page handoff under desktop headline: https://www.figma.com/file/3iOjW12ERFRDSVnpLuhQmf/landing-page-handoff?node-id=0%3A1&t=KBmvHK0K2zJCsBQK-1

- #4928 this one is also pretty important

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #4918
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-28 at 7 53 47 PM

### CODE REVIEW

**Comment by prachi00** [comment]
I havent added the dropdown yet like on figma because we dont have the url for the cookie policy yet, and I think it doesnt make sense to have a dropdown with only one option, so would be nice to have #4928 merged first before adding the dropdown

**Comment by yangwao** [comment]
Somethign is with PR deploy

image

**Comment by exezbcz** [comment]
> cookie popup is displayed on the left side of screen, in case of mobile its middle

this

- please switch the order of the button for now + make the reject all regular weight

can you please also make the mobile version? 
image

**Comment by exezbcz** [comment]
yes, good, can you please add some margin to the bottom on mobile?
image

+ use please correct hover effect, in dark mode, light mode is oki
image

**Comment by prachi00** [comment]
> yes, good, can you please add some margin to the bottom on mobile? image
> 
> * use please correct hover effect, in dark mode, light mode is oki
> 
> image

done

**Comment by vikiival** [comment]
Screenshot 2023-03-31 at 12 02 02

Screenshot 2023-03-31 at 12 02 02

Left is ok

Maybe my personal opinion but I would make the cokiie baner to use 100% or at least same width as cards behind, I find this size bit distracting. WDYT @exezbcz ?

IMG_0967

File: locales/en.json [hunk 1]
```diff
@@ -1007,7 +1007,7 @@
     "signStuckText": "Stuck? Make sure to check for any popups and SIGN the transaction."
   },
   "cookies": {
-    "notice": "We use third-party 🍪 in order to personalize your site experience",
+    "notice": "We use cookies for better service",
```
Comment (vikiival) [inline locales/en.json:1010]:
```suggestion
    "notice": "We use 🍪 for better service",
```

**Review by vikiival — APPROVED** [review]
LGTM

**Comment by exezbcz** [comment]
@vikiival yes, I think once we have the style with the more options we can stretch the cookie popup accordingly on the mobile 

For now It works, at least we begin to gather some data

**Comment by vikiival** [comment]
pay 25 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $25 @ 34.19 USD/KSM ~ 0.731 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x3f3d66385ec86bc3fb03775c7a7a106b4a21e50a957ac16d353a19211800829d](https://kusama.subscan.io/extrinsic/0x3f3d66385ec86bc3fb03775c7a7a106b4a21e50a957ac16d353a19211800829d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**